samples/bluetooth/broadcast_audio_source/prj.conf

```diff
@@ -13,3 +13,10 @@ CONFIG_BT_ISO_TX_BUF_COUNT=4
 CONFIG_BT_ISO_TX_MTU=60
 
 CONFIG_BT_DEVICE_NAME="Broadcast Audio Source"
+
+# program stack and CPU usage
+CONFIG_THREAD_ANALYZER=y
+CONFIG_THREAD_ANALYZER_USE_PRINTK=y
+CONFIG_THREAD_ANALYZER_AUTO=y
+CONFIG_THREAD_ANALYZER_AUTO_INTERVAL=5
+CONFIG_THREAD_NAME=y
```

Comment:
This should not be enabled by default in the sample IMO

Reply:
Removed